research-software-directory/RSD-as-a-service
# Refactor repository url table

# Refactor repositiry_url table

Changes proposed in this pull request:

* Merge the `programming_languages` table into the `repositiry_url` table
* Only allow one repository url per software
* For existing software that had multiple repo urls, the extra ones are moved to the description in the migration script 

How to test:

* The database structure has changed, so we need to rebuild everything:
* `docker-compose down --volumes`
* `docker-compose build`
* `docker-compose up`
* Run the data migration script
* Every two minutes, one of the scrapers should run. 
* Check the following to see if data is scraped:
	* http://localhost/api/v1/repository_url?order=license_scraped_at
	* http://localhost/api/v1/repository_url?order=languages_scraped_at
	* http://localhost/api/v1/repository_url?order=commit_history_scraped_at

PR Checklist:
